pkg/models/v1/models.go

```diff
@@ -223,10 +223,10 @@ func GetVASPEmailLog(vasp *pb.VASP) (emails []*EmailLogEntry, err error) {
 }
 
 // Normalize the email and convert to bytes
-func ContactKey(email string) []byte {
+func NormalizeEmail(email string) string {
```

Comment:
Is there a reason why this wouldn't be the same method as in the trtl store?

Reply:
The leveldb store requires a string to add a byte prefix to using the makeKey method with the preContacts string as an argument, while the trtl requires a byte key, I pulled out the common code into NormalizeEmail but it turns out that the last step is unfortunately different for each store.

Reply:
Gotcha, I definitely like having `NormalizeEmail` as a shared method. The only suggestion I would have is to move the call to `NormalizeEmail` to the `contactKey` method, which would simplify the code in the store methods.

Reply:
That's a really good idea! I just implemented that and it's much cleaner.